preprocess_data.R

```diff
@@ -30,7 +30,28 @@ if (!is.na(parsed_args$config)) {
 # Will want to put these in functions but for now just do everything
 # in a script
 
-raw_data <- read.csv(config$data_url[index])
+raw_data_all <- read.csv(config$data_url[index])
+# hardcoding this in for now bc can't get filtering from variables to work
+# see below
+
+# this_raw_data <- raw_data |>
+#   filter(location %in%
+#     config$locs_list[[{{config$regions_to_fit[index]}}]])\
+
+if (config$regions_to_fit[index] == "NYC") {
+  raw_data <- raw_data_all |>
+    filter(location %in%
```

Comment:
Note for the future single function pipes are a listing offence

Reply:
Ah right I forgot this :( will modify